<launch>
    <include file="$(find realsense2_camera)/launch/rs_camera.launch"/>
    <node name="pcan_reader" pkg="nrf_project" type="pcan_reader" output="screen"/>
    <node name="pcan_decoder" pkg="nrf_project" type="pcan_decoder.py" output="screen"/>
    <node name="local_map_publisher" pkg="nrf_project" type="local_map_publisher" output="screen"/>
    <node name="il_ins" pkg="inertiallabs_ins" type="il_ins"/>
</launch>